changelog/fix_better_offense_message_root_pathname_methods.md

```diff
@@ -0,0 +1 @@
+* [#1345](https://github.com/rubocop/rubocop-rails/issues/1345): Improve offense message for `Rails/RootPathnameMethods`. ([@earlopain][])
```

Comment:
Please remove the changelog entry since the behavior of the cop does not change.